src/components/menu/Menu.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useOutsideClickHook } from "../../hooks/useOutsideClick";
+import { useAtom } from "jotai";
+import { isMenuOpenAtom } from "../App";
+import { Island } from "../Island";
+import MenuItem from "./MenuItem";
+import MenuButton from "./MenuButton";
+import MenuSeparator from "./MenuSeparator";
+import * as MenuComponents from "./MenuComponents";
```

Comment:
let's rename `MenuComponents` to `MenuDefaultItems`

Reply:
Further, I still thing all the default compos should be called `Menu.Item*` so there's no namespace issues in the future, but also so that it's clear what's an item and what a container or other meta components. And you can type `Menu.Item` for the intellisense to list you all the items.